src/plugins/template/tests/functional/op_reference/multinomial.cpp

```diff
@@ -0,0 +1,268 @@
+// Copyright (C) 2022 Intel Corporation
```

Comment:
Just double-check if this implementation is aligned with both PyTorch and TensorFlow Multinomial operation?

Reply:
Of course, I have to enable the remaining 2 arguments and then it should be equivalent. I will make sure to double check!

Reply:
Checked, it should match both of them!